capstone/src/main/webapp/index.html

```diff
@@ -14,24 +14,20 @@
     <script src="script.js"></script>
     <script type="module" src="style.js"></script>
   </head>
-  <body onload="getRestaurants()" class="mdc-typography">
+  <body onload="setHomePage()" class="mdc-typography">
     <aside class="mdc-drawer mdc-drawer--modal">
       <div class="mdc-drawer__header">
-        <h3 class="mdc-drawer__title">EipCURE</h3>
+        <h3 class="mdc-drawer__title">EpiCURE</h3>
       </div>
       <div class="mdc-drawer__content">
         <div class="mdc-list">
-          <a class="mdc-list-item mdc-list-item--activated" href="#" aria-current="page" tabindex="0">
-            <i class="material-icons mdc-list-item__graphic" aria-hidden="true">inbox</i>
-            <span class="mdc-list-item__text">Inbox</span>
+          <a id="login-button" class="mdc-list-item mdc-list-item--activated" href="#" aria-current="page" tabindex="0">
+            <i id="login-button-icon" class="material-icons mdc-list-item__graphic" aria-hidden="true">login</i>
```

Comment:
I might be mistaken about this, but I think this tag was only for icons. So unless we use icons, this can probably be taken out.

Reply:
Yeah, I was playing around with making it a button for the home image instead of an icon but the formatting always seemed to be off or the button was the wrong color. I'll try it out again though!

Reply:
Oh yes, this tag actually makes a login icon that looks pretty good!